DfE.FindInformationAcademiesTrusts/Pages/Trusts/Academies/AcademiesAreaModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿using DfE.FindInformationAcademiesTrusts.Configuration;
+using DfE.FindInformationAcademiesTrusts.Data;
+using DfE.FindInformationAcademiesTrusts.Pages.Trusts.Academies.InTrust;
+using DfE.FindInformationAcademiesTrusts.Pages.Trusts.Academies.Pipeline;
+using DfE.FindInformationAcademiesTrusts.Services.Academy;
+using DfE.FindInformationAcademiesTrusts.Services.DataSource;
+using DfE.FindInformationAcademiesTrusts.Services.Export;
+using DfE.FindInformationAcademiesTrusts.Services.Trust;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement;
+
+namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts.Academies;
+
+public abstract class AcademiesAreaModel(
+    IDataSourceService dataSourceService,
+    ITrustService trustService,
+    IAcademyService academyService,
+    IExportService exportService,
+    ILogger<AcademiesAreaModel> logger,
+    IDateTimeProvider dateTimeProvider,
+    IFeatureManager featureManager
+) : TrustsAreaModel(dataSourceService, trustService, logger)
+{
+    private readonly IFeatureManager _featureManager = featureManager;
+
+    public override TrustPageMetadata TrustPageMetadata =>
+        base.TrustPageMetadata with { PageName = ViewConstants.AcademiesPageName };
+
+    internal readonly IAcademyService AcademyService = academyService;
+
+    protected IExportService ExportService { get; } = exportService;
+    public IDateTimeProvider DateTimeProvider { get; } = dateTimeProvider;
+
+    public AcademyPipelineSummaryServiceModel PipelineSummary { get; set; } = default!;
+    public string TrustReferenceNumber { get; set; } = default!;
+
+    public List<TrustTabNavigationLinkModel> TabList { get; set; } = [];
+
+    public override async Task<IActionResult> OnGetAsync()
+    {
+        var pageResult = await base.OnGetAsync();
+
+        if (pageResult.GetType() == typeof(NotFoundResult)) return pageResult;
+        TrustReferenceNumber = await AcademyService.GetAcademyTrustTrustReferenceNumberAsync(Uid);
+
+        PipelineSummary = await AcademyService.GetAcademiesPipelineSummaryAsync(TrustReferenceNumber);
```

Comment:
This isn't behind a feature flag so it will call the new db tables, is the database ready for us on Prod?